about.htm

```diff
@@ -0,0 +1,134 @@
+          <li class="link"><a id="border" href="">LL Coding</a></li>
+        </ul>
+      </div>
+    </div>
+
+      <i class="fas fa-bars fa-3x"></i>
+      <nav id="menu" class="invisible">
+        <ul class="menu-ul">
+            <li><i class="fal fa-times fa-3x cancel-icon"></i></li>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="#About">Join</a></li>
+            <li><a href="#Contact">Sponsor</a></li>
+            <li><a href="#Contact">News</a></li>
+            <li><a href="#Contact">LL Coding</a></li>
+        </ul>
+    </nav>
+    </nav>
+  </header>
+
+  <main>
+    <section class="section-1">
+      <h2>"Hello World!"</h2>
+      <h2>LET'S LEARN CODING TOGETHER</h2>
+      <p>LL Coding Together is a non-profit organization that consists of an interactive learning web platform, an online community forum, chat rooms, online publications and local organizations that intend to make learning web development accessible to anyone.</p>
+      <article>
+        <p>Please contact us per Email for further questions about LL Coding Together.<span class="new bold">abebeb@gmail.com</span></p>
+      </article>
+
+    </section>
+  </main>
+</div>
+
+  <section class="section-2">
+    <h2>Let's Learn Coding Together Logo</h2>
+    <hr>
+    <p>The logo of LL Coding Together was decided through the logo competition that took place from June 8 to July 7.<span class="desktop">We have selected this logo because it got a lot of votes on our platform.</span></p>
+  </section>
+
+  <section class="section-3">
+    <i class="fas fa-code fa-5x"></i>
+    <div>
+      <h2>Let's Learn Coding Together</h2>
+      <p>Addis Ababa, Ethiopia, Africa</p>
+    </div>
+  </section>
+
+  <section class="section-4">
+    <h2>See the past LL Coding Together</h2>
+    <hr>
+    <p>Take a look at the last LL Coding Together in Asia and Europe.</p>
+   
+    <div class="past-container">
+      <a href="#" class="past-link">
+
+        <div class="past asia">
+          
+          <h2>2017</h2>
+          <p>LL Coding Together in Asia</p>
+        </div>
+      </a>
+      <a href="#" class="past-link">
+
+        <div class="past europe">
+          
+          <h2>2019</h2>
+          <p>LL Coding Together in Europe</p>
+        </div>
+      </a>
+      
+    </div>
+  </section>
+
+
+  <section class="section-5">
+    <h2>Partners</h2>
+    <hr>
+    <ul>
+      <li><i class="fab fa-firefox"></i>Mozilla</li>
+      <li><i class="fab fa-google"></i>Google</li>
+      <li><i class="fab fa-free-code-camp"></i>FCC</li>
+      <li><i class="fab fa-microsoft"></i>Microsoft</li>
+      <li><i class="fab fa-airbnb"></i>Airbnb</li>
+    </ul>
+  </section>
+
+  <footer>
+    <div class="div-1">
+      <i class="footer-icon fas fa-code fa-3x"></i>
+      <h3>Let's Learn Coding Together</h3>
+    </div>
+
+    <div class="div-2">
+      <p>2021 Let's Code Together.</p>
+      <p>Some Rights Reserved.</p>
+    </div>
+  </footer>
```

Comment:
The changes to the footer section need further improvement.
The **background color** of your footer on both the mobile and desktop versions are the same. As specified let's ensure that:
- [ ]  The footer, on mobile size should have a white background, which is implemented.
- [ ] The footer, on desktop size should have a dark background.

Reply:
kindly specify the page. is it the about page or the index page?